run/calling_dash.py

```diff
@@ -160,6 +161,7 @@ def main():
     ]
 
     # Main content of streamlit app
+    st.write(f"## Patient {st.session_state['name_idx'] + 1}/{len(patient_ids)}")
```

Comment:
I'm going to leave this for now, this would require some html and setting unsafe_html to true and also means I have to center the buttons in the columns below